ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -367,3 +367,23 @@ def create_cluster(self, name: str, user: str, secret: str, nodeid: str) -> dict
                 raise ClusterManagerError("Cluster is not started.")
         except Exception as e:
             raise ClusterManagerError(f"Failed to create cluster. Error: {e}")
+
+    @controller_error_handler
+    def destroy_cluster(self, retry_index: int = 0):
```

Comment:
Need force options also

if force is true it will kill cluster and destroy immediately

Reply:
Done

Reply:
if force true
```
pcs cluster kill
pcs cluster destroy
```
kill is missing

Reply:
Ok. So, first cluster stop will happen. If that stop gets successful, there will be only cluster destroy not kill. And if cluster stop is unsuccessful for some reason, then kill + destroy (if force option is true).